js-intermediate.js

```diff
@@ -62,11 +104,35 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
+const middleLet = (string) => {
+	let letterArr = string.split("")
+	if (letterArr.length % 2 !== 0) {
+		return onlyMiddle = letterArr.slice(letterArr.length / 2, letterArr.length / 2 + 1)
+	}else {
+		return onlyMiddle = letterArr.slice(letterArr.length / 2 - 1, letterArr.length / 2 + 1)
+	}
+}
+console.log(middleLet(middleLetters1));
+console.log(middleLet(middleLetters2));
 
 
```

Comment:
Functionality looks great. Right now you are returning an array. Adding a join will get you back to the expected string.